doofinder-for-woocommerce/includes/class-rest-api-handler.php

```diff
@@ -58,9 +58,17 @@ public static function add_woocommerce_product_images($result,  $server,  $reque
         ) {
             $products = $result->data;
             foreach ($products as $key => $product) {
+                if (!is_array($product))
```

Comment:
According to WP coding standards, curly braces are always required even for on-line statements: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#brace-style :smiley: